src/backend/generator.js

```diff
@@ -314,7 +314,7 @@ exports.createDistDir = function(req, socket, callback) {
           distHelper.optimizeLogo(logoPath, socket, function(err, pad) {
             if(err) {
               console.log(err);
-              return done(err);
+              //return done(err);
```

Comment:
@agbilotia1998 This is not the correct solution here, seems like the `logoPath` is missing. Please handle the case when there is no logo instead of commenting the code